cabal-install/src/Distribution/Client/ProjectBuilding.hs

```diff
@@ -559,6 +562,7 @@ invalidatePackageRegFileMonitor PackageFileMonitor{pkgFileMonitorReg} =
 -- It requires the 'BuildStatusMap' gathered by 'rebuildTargetsDryRun'.
 --
 rebuildTargets :: Verbosity
+               -> ProjectConfig
```

Comment:
This is a breaking change since `rebuildTargets` is exported https://hackage.haskell.org/package/cabal-install-3.8.1.0/docs/Distribution-Client-ProjectBuilding.html#v:rebuildTargets